### PR TITLE
BusUpdate201711090926

### PR DESCRIPTION
**Pull request by brazanet**
small error fix on both bus/pt-ml-ae.html

### CODE REVIEW

File: stationkb/en/connections/bus/pt-ml-ae.html [hunk 1]
```diff
@@ -35,4 +35,4 @@
     <ul>
         <li><b>1</b> City Centre</li>
     </ul>
-</p
\ No newline at end of file
+</p
```
Comment (gbl08ma) [inline stationkb/en/connections/bus/pt-ml-ae.html]: The `</p>` is not correct here

File: stationkb/pt/connections/bus/pt-ml-ae.html [hunk 2]
```diff
@@ -35,4 +35,4 @@
     <ul>
         <li><b>1</b> City Center</li>
     </ul>
-</p
\ No newline at end of file
+</p
```
Comment (gbl08ma) [inline stationkb/pt/connections/bus/pt-ml-ae.html]: The `</p>` is not correct here either

Also, what did you change exactly?

Comment (brazanet) [reply stationkb/pt/connections/bus/pt-ml-ae.html]: <li><b>705</b> Estação Oriente (Interface) ⇄ Estação Roma-Areeiro</li>
 -		<li><b>708</b> Martim Moniz ⇄ Parque das Nações Norte</li>
 -		<li><b>717</b> Praça do Chile ⇄ Fetais</li>
 +	<li><b>708</b> Martim Moniz ⇄ Parque das Nações Norte</li>
this

Comment (brazanet) [reply stationkb/pt/connections/bus/pt-ml-ae.html]: i just put the lines aligned

Comment (gbl08ma) [reply stationkb/pt/connections/bus/pt-ml-ae.html]: oh ok, then take the opportunity to fix the HTML tags which are missing the `>` and I'll merge this

(just add more commits to your master branch)

**Pull request merged**